spec/controllers/idv/capture_doc_controller_spec.rb

```diff
@@ -23,107 +23,52 @@
   end
 
   describe '#index' do
-    context 'document capture step disabled' do
-      token = nil
-      before do
-        allow(FeatureManagement).to receive(:document_capture_step_enabled?).and_return(false)
-        capture_doc = CaptureDoc::CreateRequest.call(user.id, {})
-        token = capture_doc.request_token
-      end
-
-      context 'with no token' do
-        it 'redirects to the root url' do
-          get :index
-
-          expect(response).to redirect_to root_url
-        end
-      end
-
-      context 'with a bad token' do
-        it 'redirects to the root url' do
-          get :index, params: { token: 'foo' }
-
-          expect(response).to redirect_to root_url
-        end
-      end
-
-      context 'with an expired token' do
-        it 'redirects to the root url' do
-          Timecop.travel(Time.zone.now + 1.day) do
-            get :index, params: { token: token }
-          end
-
-          expect(response).to redirect_to root_url
-        end
-      end
-
-      context 'with a good token' do
-        it 'redirects to the first step' do
-          get :index, params: { token: token }
-
-          expect(response).to redirect_to idv_capture_doc_step_url(step: :mobile_front_image)
-        end
-      end
+    session_uuid = nil
+    before do
+      document_capture_session = DocumentCaptureSession.create!(requested_at: Time.zone.now)
+      session_uuid = document_capture_session.uuid
+    end
```

Comment:
using a local variable in specs is weird to me, I think we're better off with a `let!` or an `@-var`:

either:

```suggestion
    let!(:session_uuid) do
      DocumentCaptureSession.create!(requested_at: Time.zone.now).uuid
    end
```

or

```suggestion
    before do
      document_capture_session = DocumentCaptureSession.create!(requested_at: Time.zone.now)
      @session_uuid = document_capture_session.uuid
    end
```